Assets/Scripts/Cristopher's/ChooseFactionScript1.cs

```diff
@@ -0,0 +1,72 @@
+﻿using UnityEngine;
+using UnityEngine.UI;
+
+public class ChooseFactionScript1 : MonoBehaviour {
+
+    private string firstPlayersName;
+    public InputField inputName;
+
+    public bool isClicked;
+    public Button startGame;
+
+    private Faction firstFaction;
+
+    void Start()
+    {       
+        startGame.enabled = false;
+        isClicked = false;
+    }
+
+    void Update()
+    {
+            switch (isClicked)
```

Comment:
please change switch case into if statement for boolean value

Reply:
Done!